encoding/backend.go

```diff
@@ -10,7 +10,9 @@ import (
 type BackendType string
 
 const (
+	// GnarkBackend is the default CPU backend, using the gnark-crypto library.
 	GnarkBackend  BackendType = "gnark"
+	// IcicleBackend is a GPU-accelerated backend using the icicle library.
```

Comment:
nit: It's technically both CPU and GPU backend and there's a flag that controls whether or not you use the GPU.

Reply:
[a596ea6](https://github.com/Layr-Labs/eigenda/pull/2081/commits/a596ea644c37d9640c3591e0d7cd29c2d36a6e0b)